src/screens/WebsocketResearchScreen/SampleChatScreen.tsx

```diff
@@ -45,11 +45,19 @@ export const styles = StyleSheet.create({
 
 const SampleChatScreen = () => {
   const flatlistRef = React.useRef<FlatList>();
-  const {selectedChannel, chats, goBackFromChatScreen, goToChatInfoScreen, sendChat} =
-    useChatScreenHook(ANONYMOUS);
+  const {
+    selectedChannel,
+    chats,
+    goBackFromChatScreen,
+    goToChatInfoScreen,
+    sendChat,
+    updateChatContinuity
+  } = useChatScreenHook(ANONYMOUS);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `updateChatContinuity` function is being used here, but it's not clear what this function does from the provided context. It's important to ensure that this function doesn't have any side effects or doesn't mutate the `chats` state directly as it could lead to unexpected behavior. If it does, consider using a more functional approach or updating the state properly.



<!-- This is an auto-generated comment by OSS CodeRabbit -->